src/mqt/problemsolver/qubo_tools/pathfinder/README.md

```
@@ -0,0 +1,73 @@
+This submodule of the `qubo_tools` is responsible for the QUBO formulation of pathfinding problems.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)